docs/tsp/typeServerProtocol.ts

```diff
@@ -115,43 +120,45 @@ export namespace TypeServerProtocol {
         // Name of the module the type comes from
         moduleName: string | undefined;
 
-        // Name of the type.
+        // Simple name of the type. For example, for a class `MyClass` in module `my_module`, this would be `MyClass`.
         name: string;
 
         // Flags specific to the category
         categoryFlags: number;
```

Comment:
what flags are possible here?

Reply:
FunctionFlags if a function and ClassFlags if a class. I should make that more explicit in the comment here.